.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         python-version: [ "3.11" ]
         include:
           - os: ubuntu-latest
-            python-version: "3.7"
+            python-version: "3.8"
```

Comment:
In fact, L22 and L23 can just be deleted. The CI currently works so that each older Python version is tested on one OS, and the latest Python is tested on all three OS's. Python 3.8 is already being tested on L24 and L25, so these lines could just be removed.

Reply:
You are right. Thanks for reviewing the changes and clarifying the test procedure with older Python versions. :)